sympy/solvers/tests/test_inequalities.py

```diff
@@ -284,6 +279,11 @@ def test_solve_univariate_inequality():
         Or(And(-oo < x, x < 1), And(S(1) < x, x < 2))
 
 
+def test_solve_univariate_inequality_errors():
+    raises(NotImplementedError, lambda: isolve(sin(x) < S.Half, x))
```

Comment:
Looks like there is a duplicate test in `test_solvers/test_inequalities` -- the last line of that test function.